run.sh

```diff
@@ -1,12 +1,13 @@
 #!/bin/bash
 
-JAVA_OPTS="-Djavax.net.ssl.trustStore=/app/trusted.jks -Djavax.net.ssl.trustStorePassword=${TRUST_STORE_PASSWORD} -Djavax.net.ssl.trustStoreType=jks"
+JAVA_OPTS="-Djavax.net.ssl.trustStore=/app/trusted.jks \
+           -Djavax.net.ssl.trustStorePassword=${TRUST_STORE_PASSWORD} \
+           -Djavax.net.ssl.trustStoreType=jks \
+           -Dcom.sun.management.jmxremote.local.only=false \
+           -Ddelius.endpoint.url=${DELIUS_ENDPOINT_URL} \
+           -Ddelius.api.username=${DELIUS_API_USERNAME} \
+           -DAPPLICATION_INSIGHTS_IKEY=${APP_INSIGHTS_KEY} \
```

Comment:
I think this is confusing.  None of the other apps specify the insights key in their `run.sh`. Since the environment variable will be set in the docker container anyway there is no need to pass it in here

Reply:
I think its explicitly declaring what values are being passed from the environment to the JVM which is arguably one more line than necessary but adds clarity for a simple soul like me.

Reply:
would be better to therefore call it `APPLICATION_INSIGHTS_IKEY` instead of `APP_INSIGHTS_KEY`, since when it is converted into an automatic deployment it will be the only app with a different environment variable and I presume if someone creates the environment with `APPLICATION_INSIGHTS_IKEY` it won't won't work since it will be overridden in `run.sh` :disappointed: